ani-cli

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # version number
-VERSION="1.6.0"
+VERSION="1.7.0"
```

Comment:
You should bump to 1.6.1
I can understand why that may be confusing tho

Reply:
we should have a guideline for version bumping

Reply:
I'm not sure how we should further clarify the version bump guideline.
If you have a good idea to expand 2. to be more actionable, please open another PR